AbstractCommand - assert accept only instanceof EntityManager

**AbstractCommand, line 56**

Accept only EntityManager instances:

```php
assert($em instanceof EntityManager);
```

Should be:

```php
assert($em instanceof EntityManagerInterface);
```

Same here. Looks like this happens since v2.8.2, as v2.8.1 doesn't throw `Fatal error: Uncaught AssertionError: assert($em instanceof EntityManager)`.

--- a/lib/Doctrine/ORM/Tools/Console/Command/SchemaTool/AbstractCommand.php
+++ b/lib/Doctrine/ORM/Tools/Console/Command/SchemaTool/AbstractCommand.php
@@ -20,7 +20,7 @@
 
 namespace Doctrine\ORM\Tools\Console\Command\SchemaTool;
 
-use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\EntityManagerInterface;
 use Doctrine\ORM\Tools\SchemaTool;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
@@ -53,7 +53,7 @@ abstract class AbstractCommand extends Command
         $emHelper = $this->getHelper('em');
 
         $em = $emHelper->getEntityManager();
-        assert($em instanceof EntityManager);
+        assert($em instanceof EntityManagerInterface);
 
         $metadatas = $em->getMetadataFactory()->getAllMetadata();
 

